token-lending/program/src/error.rs

```diff
@@ -125,6 +125,17 @@ pub enum LendingError {
     /// Token burn failed
     #[error("Token burn failed")]
     TokenBurnFailed,
+
+    // 35
+    /// Invalid obligation collateral amount
+    #[error("Invalid obligation collateral amount")]
+    InvalidObligationCollateral,
+    /// Obligation collateral is below minimum
+    #[error("Obligation collateral is below minimum")]
+    ObligationCollateralBelowMinimum,
+    /// Obligation collateral cannot be withdrawn to below minimum
+    #[error("Obligation collateral cannot be withdrawn to below minimum")]
+    ObligationCollateralWithdrawMinimum,
```

Comment:
Is it worth thinking about these with respect to https://github.com/solana-labs/solana-program-library/issues/993 or another other issues? I want to make sure the errors are sufficiently descriptive and non-overlapping with other obligation collateral changes

Reply:
I think the current descriptions are sufficient but can always update if needed when implementing that issue